frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/topology/components/nodeStatusIcons.css

```
@@ -21,6 +21,15 @@
   fill: #e0182d;
 }
 
+.sync {
+  color: #0066cc !important;
+  fill: currentColor !important;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)